packages/cli/src/commands/up.ts

```diff
@@ -51,14 +52,26 @@ export default class Up extends MachineCreationDriverCommand<typeof Up> {
       insecureSkipVerify: flags['insecure-skip-verify'],
     }
 
-    const { hostKey, clientId, rootUrl } = await tunnelServerHello({
-      tunnelingKey,
-      knownServerPublicKeys: pStore.knownServerPublicKeys,
-      tunnelOpts,
-      log: this.logger,
+    let clientId = ''
+    let rootUrl = ''
+    let hostKey: Buffer = Buffer.from('')
```

Comment:
no need to use `let` here. `withSpinner` returns whatever its given function returns, so you could return the result of `tunnelServerHello`.